config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
+ root "items#index"
  resources :items
  resources :tags, only: [:index, :show]
+ resources :users, only: [:new, :create]
 
+ get "/dashboard" => "users#dashboard"
+ get "/login" => "users#loginpage"
```

Comment:
Here also, why do we need `users#loginpage` instead of the restful `#new`?

Reply:
I think sticking to the RESTful 'new' might be better here, too. Otherwise we'll need to make a view to correspond to this action.